packages/flet/lib/src/utils/responsive.dart

```diff
@@ -21,7 +21,7 @@ Map<String, double> parseResponsiveNumber(
 }
 
 Map<String, double> responsiveNumberFromJson(Map<String, dynamic> json) {
-  return json.map((key, value) => MapEntry(key, parseDouble(value)));
+  return json.map((key, value) => MapEntry(key, parseDouble(value, 0)!));
```

Comment:
Why do we need explicit default values now? For clarity?

Reply:
Yep!
And also/mainly because the function now returns `null` as default, instead of `0`.